src/app/components/item-stats/item-stats.component.ts

```diff
@@ -2,6 +2,7 @@ import { Component, computed, input } from '@angular/core';
 import { getItemStat } from '../../helpers';
 import { EquipmentItemDefinition, StatBlock } from '../../interfaces';
 import { MarkerStatComponent } from '../marker-stat/marker-stat.component';
+import { rarityItemColor } from '../../helpers/item';
```

Comment:
this import should be merged with the helper above (everything should import from helpers) but it's not a big enough problem to hold this on